server/tests/2-recipes.js

```diff
@@ -673,20 +675,20 @@ describe('/GET all Recipes Test', () => {
       .set('Accept', 'application/json')
       .set('x-access-token', token)
       .end((err, res) => {
-        expect(res.statusCode).to.equal(200);
+        expect(res.statusCode).to.equal(404);
         done();
       });
   });
 });
 
 describe('/GET all Recipes by User Test', () => {
-  it('should return an array of Recipes', (done) => {
+  it('should return an empty array of Recipes', (done) => {
     chai.request(server)
       .get('/api/v1/users/myRecipes')
       .set('Accept', 'application/json')
       .set('x-access-token', token)
       .end((err, res) => {
-        expect(res.statusCode).to.equal(200);
+        expect(res.statusCode).to.equal(404);
         done();
```

Comment:
Ensure that what you are testing is what is expected.

Reply:
You should test for other stuff not just the status code, like the return body.